cdk/bin/cdk.ts

```diff
@@ -40,3 +41,36 @@ const env = { region: 'eu-west-1' };
 		},
 	});
 });
+
+
+/*
+          app: fastly-content-fanout
+          config: |
+            stacks:
+              - mobile
+            regions:
+              - eu-west-1
+            deployments:
+              fastly-content-fanout:
+                  type: fastly-compute
+          contentDirectories: |
+            fastly-content-fanout:
+              - pkg/fastly-content-fanout.tar.gz
+* */
```

Comment:
minor: This inline comment has a risk of become stale, and doesn't describe the entire `riff-raff.yaml` file. A unit test, asserting on the return of [`toYAML`](https://github.com/guardian/cdk/blob/2bffe658ee27db739d5b0fe798604ac6057a2053/src/riff-raff-yaml-file/index.ts#L275-L294), might be better? [This is what `guardian/cdk` does](https://github.com/guardian/cdk/blob/main/src/riff-raff-yaml-file/index.test.ts).

Reply:
(or drop the comment 😄)

Reply:
oops, left that in from when I was migrating, so will drop the comment and will also look to add a snapshot test since its nice for future PRs to show a realistic diff of the riff-raff YAML. thanks for the spot and suggestion...

Reply:
added snapshotting in https://github.com/guardian/fastly-content-fanout/pull/6/commits/fb2d2f0e3ca1224f3c0520a2d0376198e7687e68 (with a bit of hack to the `outdir` because I didn't want to change `cdk.ts` to be wrapped in a function which takes an `outdir` override, as I don't love major refactoring of main code for benefit of testing) - having a snapshot here is great, thanks again for the suggestion